alpha/lib/model/UserEntryPeer.php

```diff
@@ -19,6 +19,23 @@ class UserEntryPeer extends BaseUserEntryPeer {
 	protected static $class_types_cache = array(
 	);
 
+	/* (non-PHPdoc)
+	 * @see BaseUserEntryPeer::setDefaultCriteriaFilter()
+	 */
+	public static function setDefaultCriteriaFilter()
+	{
+	    // when session is not admin, allow access to user's userEntries only
```

Comment:
This method should always create a default criteria, so place the creation of the criteriaFilter and the setFilter outside the if statement.